### ISSUE TITLE
Internet not found on Arch Linux

### ISSUE THREAD

**Issue by NeoChen1024** [doktor83/SRBMiner-Multi#59]
It says "Internet not found" while it have internet connection, even though I didn't have any firewall blocking connections.

**Comment by doktor83** [comment]
IT also can mean that none of the devfee pools could be connected. Which algorithm ?

**Comment by NeoChen1024** [comment]
It's TurtleCoin

**Comment by doktor83** [comment]
That's chukwa2, i checked the devfee pools can be connected from multiple internets. So it's either your ISP blocking the outgoing connection, or a firewall/antivirus.

**Comment by kronos2411** [comment]
I am using Manjaro and after last update i have the same issue with ethash algo. Nsfminer working fine, but SRBMiner says "Internet not found".

**Comment by afterthought325** [comment]
Same Here. Also using Manjaro. I've checked my firewall and Its not blocking nor dropping any traffic outbound.

**Comment by mailman451** [comment]
Same problem here with Manjaro and ethash/autolykos2. Another computer in the same LAN running Windows runs SRBMiner perfectly fine.

**Comment by doktor83** [comment]
i wonder why no issues on windows, ubuntu, debian, mining os'es like hiveos, mmpos..etc 
Same code running on all of them.
I don't think it's algorithm related, have you tried algo's other than ethash/autolykos2?

**Comment by doktor83** [comment]
I got an idea, do you guys have wget installed? If not, install it please and report back if it fixed the issue.

**Comment by NeoChen1024** [comment]
> I got an idea, do you guys have wget installed? If not, install it please and report back if it fixed the issue.

Nope, both curl and wget is installed and still the same

**Comment by mailman451** [comment]
> I got an idea, do you guys have wget installed? If not, install it please and report back if it fixed the issue.

Nope, wget is installed and seems to work on the console with your drive.google.com link (data downloaded starts with "gumkXET0Z1@" and ends on "7Al2IZSr;Ae??⏎").

**Comment by doktor83** [comment]
OK, im gonna download Manjaro and run in VM
